src/main/java/com/example/bookstoreweb/model/Order.java

```diff
@@ -0,0 +1,58 @@
+package com.example.bookstoreweb.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE orders SET is_deleted = TRUE WHERE id = ?")
+@SQLRestriction("is_deleted = FALSE")
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "user_id")
+    private User user;
+    @Column(nullable = false)
+    @Enumerated(EnumType.STRING)
+    private Status status;
+    @Column(nullable = false)
+    private BigDecimal total;
+    @Column(name = "order_date",
+            nullable = false)
+    private LocalDateTime orderDate;
+    @Column(name = "shipping_address")
+    private String shippingAddress;
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "order")
+    private Set<OrderItem> orderItems = new HashSet<>();
```

Comment:
```suggestion
    private Set<OrderItem> orderItems;
```

Reply:
Here, when creating an Order object, I immediately create a new HashSet for orderitems, so that I do not create it every time when adding an order, but immediately add the necessary orderItem